docs/scenarios/library-authoring/index.md

```diff
@@ -14,5 +14,6 @@ ms.assetid: 93689fc6-eb31-4ca4-925f-f295db4083dd
 
 # Authoring Libraries for .NET Core 1.0
```

Comment:
Going to just delete this, as the entire sub-TOC is not part of the doc set anymore.